rac-experiments-v3.json

```diff
@@ -22,7 +22,8 @@
               "shardRange": {
                 "start": 0,
                 "end": 3333
-              }
+              },
+              "algorithmType": "CONSTANT"
```

Comment:
added this to the first flag (but none of the other existing ones) so we can ensure our SDKs are ok with or without it.